docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: '3.9'
+
+x-backend: &default-backend
+  image: physionet:latest
+  depends_on:
+    - db
+  stdin_open: true
+  tty: true
+  entrypoint: ./docker/dev-entrypoint.sh
+
+volumes:
+  dev_data:
+  test_data:
+  db_data:
+
+networks:
+  dev:
+  test:
+
+x-env: &default-env
+  DJANGO_SETTINGS_MODULE: physionet.settings.settings
+  ENVIRONMENT: development
+  SECRET_KEY: asdf
+  ALLOWED_HOSTS: localhost
+  DB_USER: physionet
+  DB_PASSWORD: password
+  DB_HOST: db
+  DEBUG: 'true'
+  SITE_ID: 4
+  EMAIL_HOST: localhost
+  DEFAULT_FROM_EMAIL: localhost@localhost
+  CONTACT_EMAIL: PhysioNet Contact <contact@dev.physionet.org>
+  SERVER_EMAIL: 'PhysioNet System <root@dev.physionet.org>'
+  ADMINS_NAME: PhysioNet Technical
+  ADMINS_MAIL: technical@dev.physionet.org
+  ORCID_REDIRECT_URI: http://127.0.0.1:8000/authorcid
+  DATACITE_API_URL: https://api.test.datacite.org/dois
+  DATACITE_PREFIX:
+  MEDIA_ROOT: /data/pn-media
+  STATIC_ROOT: /data/pn-static
+
+services:
+  dev:
+    <<: *default-backend
+    build: .
+    ports:
+      - '8000:8000'
+    environment:
+      <<: *default-env
+      DB_NAME: physionet_dev
+    command: ['python', '/code/physionet-django/manage.py', 'runserver', '0.0.0.0:8000']
+    volumes:
+      - ./pyproject.toml:/code/pyproject.toml:rw
+      - ./poetry.lock:/code/poetry.lock:rw
+      - ./docker:/code/docker:rw
+      - ./physionet-django:/code/physionet-django:rw
+      - dev_data:/data:rw
+    networks:
+      - dev
+  test:
+    <<: *default-backend
+    environment:
+      <<: *default-env
+      DB_NAME: physionet_test
+    command: ['sleep']
+    volumes:
+      - ./pyproject.toml:/code/pyproject.toml:rw
+      - ./poetry.lock:/code/poetry.lock:rw
+      - ./docker:/code/docker:rw
+      - ./physionet-django:/code/physionet-django:rw
+      - test_data:/data:rw
+    networks:
+      - test
+  db:
+    image: postgres:13-alpine
+    environment:
+      POSTGRES_USER: physionet
+      POSTGRES_PASSWORD: password
+      DEV_DB: physionet_dev
```

Comment:
Please could you briefly explain the purpose of the two separate databases here? (`DEV_DB` and `TEST_DB`)

Reply:
The reason for using 2 databases is pretty simple: the test suit requires a very specific database contents (from `loaddemo`). Using a separate database for development makes it possible to freely change the development database contents (adding/deleting users, projects etc.) while ensuring that the tests will not see those changes. Because of that there is no need to run subsequent `resetdb` or `loaddemo` on the test database after making changes via `localhost:8000`.